Pairwise_potential.py

```diff
@@ -27,12 +27,13 @@ class LJ(PairwisePotential):
 
     """
     
-    def __init__(self,sigma=1.0,epsilon=1.0):
+    def __init__(self,sigma=1.0,epsilon=1.0, cutoff=2.6):
 
         self.sigma = sigma
         self.epsilon = epsilon
+        self.cutoff2 = cutoff * cutoff
     
-    def __call__(self, rij2):
+    def lennard_jones(self, rij2):
```

Comment:
We should call this `potential` and add that to the abstract class required methods.

Reply:
I don't quite get what you mean. You mean use potential instead of call?

Reply:
Call the method potential.